instrumentation/kafka-clients/src/main/java/brave/kafka/KafkaPropagation.java

```diff
@@ -0,0 +1,33 @@
+package brave.kafka;

```

Comment:
brave.kafka.clients package name..

also add package-info.java with the following:
```java
@javax.annotation.ParametersAreNonnullByDefault
package brave.kafka.clients;
```

this makes sure IDE can tell what's nullable or not